Client/src/modules/admin/CacheSettings.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <q-page class="page-padding">
+    <div class="header-with-button">
+      <h2 class="q-title">{{$tl("title")}}</h2>
+      <div class="clear"></div>
+    </div>
+    <div>
+      <div class="q-mt-lg">
+        <q-select
+          v-model="policy"
+          :label="$tl('CachePolicy')"
+          :options="PolicyTypes"
+        />
+        <br />
+        <q-input v-if="policy !== null && policy.id !== 1" v-model="cacheSettings.invalidateCacheTime" type="number" :label="$tl('CacheLifetime')" />
+      </div>
+
+      <br>
+      <div class="btn-block">
+        <q-btn icon="fas fa-plus" class="btn-send" :loading="loading"
+          no-caps :label="$tl('SaveChanges')" color="send"
+          @click="save">
+          <LoaderSent slot="loading"/>
+        </q-btn>
+      </div>
+    </div>
+  </q-page>
+</template>
+
+<script>
+import Page from "Page";
+import LoaderSent from "LoaderSent";
+
+export default {
+  name: "CacheSettings",
+  components: {LoaderSent},
+  mixins: [Page],
+  i18nPrefix: "Admin",
+  data: function() {
+    return {
+      cacheSettings: null,
+      policy: null,
+      loading: false,
+      PolicyTypes: [
+        { id: 0, label: this.$tl("AlwaysPolicy"), value: "AlwaysPolicy" },
+        { id: 1, label: this.$tl("NeverPolicy"), value: "NeverPolicy" },
+        { id: 2, label: this.$tl("CustomPolicy"), value: "CustomPolicy" }
+      ]
+    };
+  },
+  methods: {
+    async loadCurrentPolicy() {
+      await this.$store
+        .dispatch("request", {
+          url: "/Admin/AdminCacheSettings/GetCurrentCacheSettings"
```

Comment:
Do you think it could be worth to extract such values as constants?
For example, we have duplication in line 55 and 67. In case we change /Admin to something else - we will have to change this in several places. We could have several constants for such cases(names should be improved, but to get an idea):
1. AdminEndpoint = "/Admin"
2. AdminCacheSettings = AdminEndoint + "/AdminCacheSettings" (probably can change "AdminCacheSettings -> CacheSettings)
3. GetCurrentCache = AdminCacheSettings + "/GetCurrentCacheSettings"